mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -68,18 +73,110 @@ properties:
     name: 'displayName'
     description: |
       User set display name of the template.
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    description: |
+      The creation timestamp of an deidentifyTemplate. Set by the server.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    description: |
+      The last update timestamp of an deidentifyTemplate. Set by the server.
+    output: true
   - !ruby/object:Api::Type::NestedObject
     name: 'deidentifyConfig'
     required: true
     description: Configuration of the deidentify template
     properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'imageTransformations'
+        description: Treat the dataset as an image and redact.
+        exactly_one_of:
+          - deidentify_config.0.info_type_transformations
+          - deidentify_config.0.record_transformations
+          - deidentify_config.0.image_transformations
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'transforms'
+            required: true
+            description: |
+              For determination of how redaction of images should occur.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'redactionColor'
+                  description: |
+                    The color to use when redacting content from an image. If not specified, the default is black.
+                  properties:
+                    - !ruby/object:Api::Type::Double
+                      name: red
+                      description: |
+                        The amount of red in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                    - !ruby/object:Api::Type::Double
+                      name: blue
+                      description: |
+                        The amount of blue in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                    - !ruby/object:Api::Type::Double
+                      name: green
+                      description: |
+                        The amount of green in the color as a value in the interval [0, 1].
+                      default_value: 0.0
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'selectedInfoTypes'
+                  description: Apply transformation to the selected infoTypes.
+                  exactly_one_of:
+                    - transforms.0.selected_info_types
+                    - transforms.0.all_info_types
+                    - transforms.0.all_text
+                  properties:
+                  - !ruby/object:Api::Type::Array
+                    name: 'infoTypes'
```

Comment:
`infoTypes` can be required, as it is the only filed inside the object `selectedInfoTypes`

Reply:
Added `required:true` to `infoTypes`.